packages/mint-components/src/components/sqm-leaderboard/Leaderboard.stories.tsx

```diff
@@ -163,6 +163,10 @@ const defaultStyles = {
   anonymousUser: "Anonymous User",
 };
 
+const link = <a>Support</a>;
+
+const tag = "Contact" + link + "about upgrading your plan";
```

Comment:
Needs a real link, should be a template string for proper spacing between link and text